wiki/Twitter/id.md

```diff
@@ -1,18 +1,24 @@
----
-outdated: true
-outdated_since: a6c6c6b27315fdf9e68f709dab2a08b085428e3b
----
-
 # Twitter
 
-## Kami juga menge-tweet!
+osu! memiliki beberapa akun Twitter, masing-masing melayani tujuan tertentu. Sebagian besar memiliki lalu lintas sangat rendah tetapi akan membantu kamu dalam mendapatkan informasi di luar lingkungan osu!.
+
+## Layanan
+
+| Avatar | Handle | Deskripsi |
```

Comment:
```suggestion
| Avatar | Akun | Deskripsi |
```

Reply:
don't really understand why `handle` there without translation, using kind detail help people to understand